src/Application.QueryServices/Handlers/ITransactionsQueriesHandlers.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace PetProjects.MicroTransactionsApi.Application.QueryServices.Handlers
+{
+    using PetProjects.Framework.Cqrs.Queries;
+    using PetProjects.MicroTransactionsApi.Domain.Queries;
+    using PetProjects.MicroTransactionsApi.Domain.Queries.ReadModels;
+
+    public interface ITransactionsQueriesHandlers : IQueryHandlerAsync<GetTransactionByIdQuery, TransactionByIdReadModel>
```

Comment:
Do we really need an ITransactionsQueryService?

I think the purpose of the Mediator is to avoid this kind of connection class. We could call a Command or Query inside the controller.

Reply:
If we are going with this approach I would remove the query handler and make the repository calls in this layer. This would look like a ITransactionsReadModelService/Facade wtv it could be called.

Reply:
See https://github.com/the-pet-projects/mts-transactions-api/pull/6#discussion_r154509399